### DISCUSSION TITLE
GraphQL: Resource not accessible by integration

### DISCUSSION THREAD

**Discussion by ffisc** [skills/.github#150] [Q&A]
Has anyone come across this before? The contents of the file are copy-pasted from the instructions.

<img width="737" alt="image" src="https://user-images.githubusercontent.com/29397125/213184589-3ee4fcff-cd4b-4431-adda-54d357e089eb.png">

**Comment by Mahiuha** [comment]
check the following

1. Verify that you have the correct permissions to access the repository and the pull request.
2. Check that you have provided the correct repository name and pull request number in the command.
3. Make sure that you are running the latest version of the gh command-line tool.
4. It may be possible that the repository is not open for pull requests, check for the same.
5. It may also be caused by a bug in the gh command-line tool. In this case, you can check for updates to the tool or report the issue to the developers.

**Reply by ffisc** [reply]
Thank you @Mahiuha 

1. The repo is a private repo that I created and the PR was created by me too, so I assume all fine on this front.
2. Repo name doesn't seem to be required in the command. PR number seems to be a variable that the workflow picks up, I haven't provided it manually.
3. I'm not running the command on my machine but instead as a GH action, so I'm assuming GH takes care of this.
4. I've been able to manually open and merge a PR.
5. See comment below

In short, I've just copy-pasted a YAML spec for a workflow from the [documentation](https://github.com/skills/hello-github-actions#step-2-add-a-job-to-your-workflow-file), and it's not working. It seems to be [this issue](https://github.com/actions/first-interaction/issues/10). So if there's an issue, wouldn't it make sense to change the instructions so that this is not triggered?

**Reply by Mahiuha** [reply]
It sounds like you have followed the instructions correctly, but the workflow is still not working as expected. It is possible that the issue is with the GH action or the copy-pasted YAML spec. It is also possible that there is a bug in the gh command-line tool that is causing the error. I recommend checking for updates to the tool and reporting the issue to the developers if necessary. Additionally, it would be helpful to review the documentation and see if there are any known issues or troubleshooting steps that can be taken to resolve the problem.
